### ISSUE TITLE
Feature request: MacOS menu bar icon

### ISSUE THREAD

**Issue by pineappleburger** [brentsimmons/NetNewsWire#2128]
I‘d rly like a menu bar icon that displays the number of unread feeds (Like for example Vienna does). I thought I‘d just create an issue, hope that‘s alright 😁

**Comment by brentsimmons** [comment]
Earnest question — why not just use the unread count display in the Dock icon for NetNewsWire?

**Comment by pineappleburger** [comment]
I like to hide the dock so it only appears when I hocer over it. For that extra inch of available screen, I guess 😁

**Comment by milesmcclane** [comment]
This is a great idea. I almost solely use the menu bar to operate my Mac, using apps like menuX and fluid.app and spotlight.  My doc is hidden. I would very much like to see a menu bar icon option too. Would be very cool even if basic and clicking it just opened the app!

**Comment by kevinm6** [comment]
I think it could be a nice idea!
I don't know how much I personally could use it, but it seems nice and a good feature to have, in case.
Maybe with the last 5/10 unread items or the starred one.

**Comment by milanov** [comment]
Another vote for the menu bar icon idea, even if it doesn't display the unread count. It's more convenient, at least for me, and saves dock space.

**Comment by MKE1138** [comment]
I'd really like this also, for the same reasons as @pineappleburger

**Comment by jhoek** [comment]
So maybe (optionally) hiding the Dock icon and having a menu bar icon that indicates if/how many new RSS feed items exist?

**Comment by MKE1138** [comment]
Yes, exactly. Like how [baRSS](https://github.com/relikd/baRSS) handles it.

**Comment by gorbypark** [comment]
One more thumbs up for this.  I personally don't like having apps that need to be open all the time, but are used infrequently, to be in my dock.  Much prefer a menu bar icon so the app can keep running and checking for new articles.  I'd be ok with a dock icon while the app is actually "on screen" and I'm interacting with it.

**Comment by junk1tm** [comment]
Another vote for this feature, would be a nice addition 🙏

**Comment by simgust** [comment]
Hiding dock icon + adding menu bar icon would be lovely!
